prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -235,9 +50,10 @@ def smart_charging(
     # load NHTS data from function
     if veh_type.lower() == "ldv":
         newdata = data_helper.remove_ldt(data_helper.load_data(census_region, filepath))
-
-    elif veh_type.lower() == "ldt":
-        newdata = data_helper.remove_ldv(data_helper.load_data(census_region, filepath))
+    elif veh_type.lower() == "mdv":
+        newdata = data_helper.load_hdv_data("mhdv", filepath)
+    elif veh_type.lower() == "hdv":
+        newdata = data_helper.load_hdv_data("hhdv", filepath)
```

Comment:
It seems we removed the option of `veh_type` being `ldt` here, are we merging the logic of light duty vehicles and light duty trucks together?

Reply:
I'm not really seeing used at the moment, I can add it back in...

Reply:
My read on the original logic is removing LDT (`remove_ldt`) when dealing with LDV vice versa (`remove_ldv`). Just wanna make sure we are not missing anything.

Reply:
OK, I see difference (t vs. v) between the two different remove functions now...